src/components/widgets/MetadataWidget/TabWidget/TabPresentation.tsx

```diff
@@ -33,21 +32,21 @@ function TabPresentation(props: TabPresentationProps) {
           {
                 content: (
                   <>
+                    {/* TODO: Is overflow: "auto" wanted? */}
                     <div style={{ overflow: "auto"}}>
-                    {props.useLegacy == undefined || props.useLegacy
-                      ? <HierarchyWidgetDeprecated
-                        ontologyId={props.ontologyId || ((data && data.getOntologyId() !== undefined) ? data.getOntologyId() : "")}
-                        api={props.api}
-                        iri={props.iri}
-
-                      />
-                      : <HierarchyWidgetOLS
-                        api={props.api}
+                      <HierarchyWidget
+                        // backend_type and apiKey missing here. If TabWidget/ MetadataWidget shall be used for other backend types later, this has to be provided
```

Comment:
Exactly, as soon as we connect the TS4NFDI API Gateway, we must update the parameters.

Reply:
All right! But it's okay to leave them out for now, right?

Reply:
Yes. The other parts of the MetadataWidget only work with the OLS API. This will also need to be adapted later.